proto/v1/structure/languages.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package org.oppia.proto.v1.structure;
+
+import "proto/v1/structure/versions.proto";
+
+option java_package = "org.oppia.proto.v1.structure";
+option java_multiple_files = true;
+
+// Structure corresponds to LanguageProtosVersion.
+enum LanguageCode {
+  LANGUAGE_CODE_UNSPECIFIED = 0;
+  ENGLISH = 1;
+  ARABIC = 2;
+  HINDI = 3;
+  HINGLISH = 4;
+  BRAZILIAN_PORTUGUESE = 5;
+}
+
+
+// Strucutre of the single subtitled html. 
+message SubtitledHtml {
+  LanguageProtosVersion structure_version = 1;
+
+  // Id of the content whomsoever corresponds to SubtitledHtml.
+  string content_id = 2;
+
+  // Html format data of the content whomsoever corresponds to SubtitledHtml.
+  string html = 3;
+}
+
+// Strucutre of the single recorded voiceover. 
+message RecordedVoiceovers {
+  LanguageProtosVersion structure_version = 1;
+
+  // All the voiceover corresponds to the LanguageProtosVersion.
+  repeated VoiceoverContentMapping voiceover_language_mapping = 2;
+}
+
+// Structure corresponds to LanguageProtosVersion.
```

Comment:
I don't understand this comment (and similar ones). Do you know what it means?

Reply:
Chatted with @BenHenning, got more context on this:

- Each proto needs to have a documented plan for versioning. In particular, every top-level `message` in every file must have something explaining what its corresponding proto version is.
- If any changes happen in that proto structure, the version specified in the comment must be incremented.

Please could you add this to the README? Also, let's change this and similar comments to: "NOTE: Update LanguageProtosVersion if the structure of this proto is modified."

Reply:
Added in Readme and updated all the places where proto are having `proto_version` fields.